lib/ansible/module_utils/common/respawn.py

```diff
@@ -58,7 +58,8 @@ def probe_interpreters_for_module(interpreter_paths, module_name):
     be returned (or ``None`` if probing fails for all supplied paths).
     :arg module_name: fully-qualified Python module name to probe for (eg, ``selinux``)
     """
-    env = os.environ | {'PYTHONPATH': _ANSIBLE_PARENT_PATH}
+    PYTHONPATH = os.getenv('PYTHONPATH', '')
+    env = os.environ | {'PYTHONPATH': f'{_ANSIBLE_PARENT_PATH}:{PYTHONPATH}'.rstrip(': ')}
```

Comment:
Is the space needed for the rstrip?

Reply:
It was purposefully added, but maybe me being overly cautious.  I was just envsioning someone errantly setting `PYTHONPATH` to a space, but in the end I suppose it doesn't matter.